api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -255,6 +261,10 @@ public State getState() {
         return state;
     }
 
+    void eagerInit(@Observes @Initialized(ApplicationScoped.class) Object event) {
+        // init
+    }
+
```

Comment:
The ```@Eager``` annotation once again was causing problems. Until eager initialization makes its way into CDI, this is the way to do it.